pkg/controllers/execution/execution_controller.go

```diff
@@ -219,7 +219,7 @@ func (c *Controller) syncToClusters(clusterName string, work *workv1alpha1.Work)
 
 	if len(errs) > 0 {
 		total := len(work.Spec.Workload.Manifests)
-		message := fmt.Sprintf("Failed to apply all manifests (%v/%v): %v", syncSucceedNum, total, errors.NewAggregate(errs).Error())
+		message := fmt.Sprintf("Failed to apply all manifests (%d/%d): %v", syncSucceedNum, total, errors.NewAggregate(errs).Error())
```

Comment:
```suggestion
		message := fmt.Sprintf("Failed to apply all manifests (%d/%d): %s", syncSucceedNum, total, errors.NewAggregate(errs).Error())
```
If you prefer the precise format, why not print the string by `%s`.